### PR TITLE
Apply coupon and Payment Integration

### PR DESCRIPTION
**Pull request by AtulPant2704**
- Apply coupon on cart page based on price.
- Payment Integartion using Razorpay API.

### CODE REVIEW

**Review by Kevin-Solomon — APPROVED** [review]
Clean UI. 
Functionality Works as expected.

**Review by AnkurChunekar — APPROVED** [review]
All functionalities works as intended, code looks neat and clean great work @AtulPant2704

File: src/utils/cartHandlers/removeFromCartHandler.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import { removeFromCartService } from "../../services";
 
-const removeFromCartHandler = async (_id, token, cartDispatch) => {
+const removeFromCartHandler = async (_id, token, cartDispatch, from = "") => {
   try {
     const response = await removeFromCartService(_id, token);
     if (response.status === 200) {
       cartDispatch({ type: "REMOVE_FROM_CART", payload: response.data.cart });
-      toast.info("Removed from cart");
+      !from && toast.info("Removed from cart");
```
Comment (VishalPatil18) [inline src/utils/cartHandlers/removeFromCartHandler.js]: You could use the ternary operator here, as that covers all edge cases.

**Review by VishalPatil18 — APPROVED** [review]
The code looks quite good and works well. Great work @AtulPant2704 👍

**Pull request merged**